onebusaway-android/src/main/java/org/onebusaway/android/nav/NavigationServiceProvider.java

```diff
@@ -706,4 +716,113 @@ private void silence(long duration, int queueFlag) {
             mTTS.playSilence(duration, queueFlag, null);
         }
     }
+
+    public void getUserFeedback() {
+        //TODO - Log "Yes" or "No" including plaintext feedback using Firebase Analytics
+
+        Application app = Application.get();
+        NotificationCompat.Builder mBuilder;
+
+        String message = Application.get().getString(R.string.feedback_notify_dialog_msg);
+        mFirstFeedback = Application.getPrefs().getBoolean(FIRST_FEEDBACK, true);
+
+        // Create delete intent to set flag for snackbar creation next time the app is opened.
+        Intent rIntent = new Intent(app.getApplicationContext(), FeedbackReceiver.class);
+        rIntent.putExtra(FeedbackReceiver.NOTIFICATION_ID, NOTIFICATION_ID + 1);
+
+        if ((mFirstFeedback) || (Build.VERSION.SDK_INT < Build.VERSION_CODES.N) || (Build.VERSION.SDK_INT >= Build.VERSION_CODES.P)) {
+
+            Intent fdIntent = new Intent(app.getApplicationContext(), FeedbackActivity.class);
+            fdIntent.setAction(REPLY_ACTION);
+            fdIntent.putExtra("CallingAction", "no");
+            fdIntent.putExtra("NotificationId", NOTIFICATION_ID + 1);
+            fdIntent.putExtra("TripId", mTripId);
+            PendingIntent fdPendingIntentNo = PendingIntent.getActivity(app.getApplicationContext()
+                    , 1, fdIntent,PendingIntent.FLAG_UPDATE_CURRENT);
+
+            fdIntent = new Intent(app.getApplicationContext(), FeedbackActivity.class);
+            fdIntent.setAction(REPLY_ACTION);
+            fdIntent.putExtra("CallingAction", "yes");
+            PendingIntent fdPendingIntentYes = PendingIntent.getActivity(app.getApplicationContext()
+                    , 2, fdIntent, PendingIntent.FLAG_UPDATE_CURRENT);
+
+            rIntent.putExtra(FeedbackReceiver.ACTION_NUM,
+                    FeedbackReceiver.DISMISS_FEEDBACK_NOTIFICATION);
+            PendingIntent pDelIntent = PendingIntent.getBroadcast(app.getApplicationContext(),
+                    0, rIntent, 0);
+
+            mBuilder = new NotificationCompat.Builder(Application.get().getApplicationContext()
+                    ,Application.CHANNEL_DESTINATION_ALERT_ID)
+                    .setSmallIcon(R.drawable.ic_stat_notification)
+                    .setContentTitle(Application.get().getResources()
+                            .getString(R.string.feedback_notify_title))
+                    .setContentText(message)
+                    .addAction(0, "No", fdPendingIntentNo )
```

Comment:
If these "No" and "Yes" values are user-facing in the UI, they need to be in `strings.xml` so they can be translated.

Reply:
Added "No" and "Yes" values to `strings.xml`.